example/external-config/webpack.config.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
           },
           {
             loader: "../../index.js",
+            options: {
+              configFile: './svgo.config.js'
+            }
```

Comment:
Do we need this in the example, or have you added this to check that the options a passed as expected? Should we remove it, in that case?

Reply:
Yes, I thought we need to add options into the example, and I also have checked the options are passed as expected.
# After:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/1962079/197782156-e647829e-8436-4531-a232-6c0839e4df78.png">

# Before
<img width="969" alt="image" src="https://user-images.githubusercontent.com/1962079/197783127-e46d96b8-7b6e-4460-a230-b464742f0685.png">

Reply:
Additionally, [example/basic](https://github.com/svg/svgo-loader/blob/main/example/basic/webpack.config.js) and  [example/external-config](https://github.com/svg/svgo-loader/blob/main/example/external-config/webpack.config.js) are totally the same now, so I believe use `external-config` to test the configFile option is okay.

![image](https://user-images.githubusercontent.com/1962079/197788223-bb419fd3-60a4-455a-9d04-30fd606020c1.png)

Reply:
Yeah, makes sense to have some options passed somewhere so it's tested